site/content/docs/4.3/content/tables.md

```diff
@@ -575,15 +720,106 @@ Regular table background variants are not available with the dark table, however
 {{< partial "callout-warning-color-assistive-technologies.md" >}}
 {{< /callout >}}
 
+## Nesting
```

Comment:
This needs more explanation. Also, I'm not a fan of the styles leaking to the nested table. But maybe we should have a look at this afterwards to prevent this PR from becoming too huge?

Reply:
Oh yeah, I added the nesting to show how it happens, and it basically convinced me we needed to figure that out. I'm thinking we could find a balance of styling a single first descendant?

Something like this maybe:

```scss
.table {
 > tbody  {
    td,
    th {
      ...
    }
  }
}
```

Reply:
> ```scss
>  .table {
>   > tbody  {
>      td,
>      th {
>        ...
>      }
>    }
>  }
> ```

Would still leak into nested table thead and tfoot (as well as tbody)